.github/workflows/CI.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         run: npm run lint
       
       - name: Build
-        run: npm run build 
+        run: npm run build:prod
```

Comment:
Just out of curiosity, what prompted this change?

Reply:
It was necessary to get CI to pass because otherwise it uses the default `development` environment and looks for an `environment.development.ts` file which has been gitignored due to the fact that it stores the API access token